cves/2021/CVE-2021-20038.yaml

```
@@ -0,0 +1,37 @@
+id: CVE-2021-20038
+
+info:
+  name: SonicWall SMA100 Stack BoF to Unauthenticated RCE
+  author: dwisiswant0 & wvu
```

```suggestion
  author: dwisiswant0 & Jacob Baines
```

This is @jbaines-r7's research. :)

---

Thanks!